aerogear/datasync-starter
# Use env file

## Motivation 

Use the same environment file and docker image for che and local machine.
To not pass multiple environment variables we are using env file that can simplify things. 

## Verification

1. Start docker and server
2. See url (you can also print it by console)

## Verification in Eclipse Che

See shared workspace for that
